.env.example

```diff
@@ -0,0 +1,75 @@
+# Host Ports
```

Comment:
I would propose to name this file to `docker-env.ci` and consequently use it in the entire workflow, i.e.
- remove the "Create env file" step in `check.yml`
- use that file in `docker-compose.ci.yml`'s `env_file` key.

The reason is that `.env` is too generic which might lead to ambiguities.

Reply:
Ok, I will change it.

Reply:
Aha, this approach has failed. Maybe it is compulsory to have a .env extension!